app/assets/stylesheets/book.css.scss

```diff
@@ -313,3 +313,17 @@ $cart-view-width: 250px;
   padding: 10px;
   background-color: white;
 }
+
+.project-select {
+  margin-right: 20px;
+}
+
+.group-name {
+  display: inline;
+}
+
+.group-name:hover {
+  color: #c0392b;
```

Comment:
Color literals like `#c0392b` should only be used in variable declarations; they should be referred to via variable everywhere else.